README.rst

```diff
@@ -17,7 +17,7 @@ Quickstart
     $ slack-cli -d general "Hello everyone!"
 
 
-You will be asked to provide a Slack API token. It's easy, just get one from the `API token generator <https://api.slack.com/custom-integrations/legacy-tokens>`_.
+You will be asked to provide a Slack API token. It's easy, just refer to the `docs here <docs/SLACK_APP.md>`_.
```

Comment:
Could you please move these docs to the README? I'd rather have all useful information gathered in one place.

Reply:
That's what I did initially, but then I though it'd clutter up the README too much with screenshots and stuff, like for people, who just want a quick overview.

Maybe move it to a bottom section of the README?

Reply:
> Maybe move it to a bottom section of the README?

Yes, that's a good idea.